atomic_reactor/config.py

```diff
@@ -340,6 +341,11 @@ def koji_path_info(self):
         root_url = self.koji['root_url'].rstrip('/')
         return PathInfo(topdir=root_url)
 
+    @property
+    def otel_collector_url(self):
+        return self._get_value(ReactorConfigKeys.OPENTELEMETRY_COLLECTOR_URL_KEY,
```

Comment:
we will need that to be defined in osbs-client config as well, so I'd just pass it via user_params

Reply:
moved to `user_params`